pom.xml

```diff
@@ -469,7 +469,7 @@
 				<configuration>
 					<skipProvidedScope>true</skipProvidedScope>
```

Comment:
It's a minor thing, but please fix the indentation in this section.

Reply:
checked from IDE, it's not showing an indentation issue. still reformatted the file.

Reply:
The IDE may not show it because there is a mix of spaces and tabs here for indentation. It should be consistent with other lines in the file.

Reply:
yes @hankwallace , found the spacing issue. then updated indentation with "Tab"